internal/server/translator/golden/query/statvar-obs.json

```diff
@@ -10,7 +10,7 @@
           "value": "country/ABW"
         },
         {
-          "value": "29730"
+          "value": "30320"
```

Comment:
@shifucun I can't tell from the file what SV are these?

Reply:
Can you check the value for 2000: https://autopush.datacommons.org/browser/country/ABW?statVar=Amount_EconomicActivity_GrossNationalIncome_PurchasingPowerParity_PerCapita

This changed

Reply:
Yup, confirmed that this changed in the World Bank data: https://databank.worldbank.org/reports.aspx?source=world-development-indicators (https://screenshot.googleplex.com/4MDLC3i6UdRMnW5)